ext/fileinfo/fileinfo.stub.php

```diff
@@ -92,6 +92,7 @@ public function set_flags(int $flags): true {}
 /** @refcount 1 */
 function finfo_open(int $flags = FILEINFO_NONE, ?string $magic_database = null): finfo|false {}
 
+#[\Deprecated(since: '8.5', message: 'as finfo objects are freed automatically')]
```

Comment:
```suggestion
#[\Deprecated(since: 'PHP 8.5', message: 'as finfo objects are freed automatically')]
```

Isn't it better to be explicit about the version? There may be case that the plain version number may be confused with other software, i.e. MySQL has close version number.

Reply:
This is a discussion for another place. The “bare version” format is what is currently used by PHP core. See: https://github.com/php/php-src/commit/29f98e748568ebd66aaae061c0dcefbba92ca058